Command failed: "d:/CP/Educational Codeforces Round 138 Rated for Div. 2/D_Counting_Arrays/Tests/a.exe" 1 < "d:/CP/Educational Codeforces Round 138 Rated for Div. 2/D_Counting_Arrays/Tests/input_4.txt" > "d:/CP/Educational Codeforces Round 138 Rated for Div. 2/D_Counting_Arrays/Tests/code_output_4.txt"
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
